charts/incubator/dumbassets/values.yaml

```diff
@@ -0,0 +1,39 @@
+image:
+  repository: dumbwareio/dumbassets
+  pullPolicy: IfNotPresent
+  tag: 1.0.11@sha256:b12310240508ee26acd4adbbf4b3e42303a4c5918b094daf0d225b3ae00cad93
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
```

Comment:
was this tested without root?

Reply:
Assuming the app-template is using the default truecharts permissions, it should run without root permissions. I've removed it.